chsdi/models/bod.py

```diff
@@ -125,6 +126,18 @@ def layerConfig(self, params):
         if 'attribution' in config:
             config['attributionUrl'] = translate(self.__dict__['attribution'] + '.url')
 
+        # adding __queryable_attributes__ if they have them
+        models = models_from_name(self.layerBodId)
+        if models is not None:
+            queryable_attributes = []
+            for model in models:
+                if hasattr(model, '__queryable_attributes__'):
+                    queryable_attributes.extend(model.get_queryable_attributes_keys(params.lang))
+
+            queryable_attributes = list(set(queryable_attributes))
```

Comment:
Kind of the same than https://github.com/geoadmin/mf-chsdi3/blob/gjn_attr/chsdi/views/layers.py#L112
Is is worthed to factorize a little?